website/content/specification/syntax/instances.md

```diff
@@ -228,7 +228,7 @@ The following behaviors are REQUIRED to be used for each value of `@required`.
 
 A *model instance* is used to declare a relationship to other information elements in an assembly definition's model.
 
-There are 5 kinds of model instances, which can be declared as part of the assembly's model.
+The following kinds of model instances can be declared as part of the assembly's model.
```

Comment:
There are now 6 items in the bulleted list. I reworded the lead-in sentence to make it more maintainable in case the list evolves further.